packages/IntegrationTest/windows/integrationtest/TestCommandListener.cpp

```diff
@@ -40,23 +40,21 @@ IAsyncAction TestCommandResponse::TestFailed() noexcept {
   co_await SendJson(responseJson);
 }
 
-IAsyncAction TestCommandResponse::Exception(const IRedBoxErrorInfo &err) noexcept {
+IAsyncAction TestCommandResponse::Exception(const ExceptionInfo &err) noexcept {
   JsonObject responseJson;
   responseJson.SetNamedValue(L"status", JsonValue::CreateStringValue(L"exception"));
-  responseJson.SetNamedValue(L"message", JsonValue::CreateStringValue(err.Message()));
-  responseJson.SetNamedValue(L"originalMessage", JsonValue::CreateStringValue(err.OriginalMessage()));
-  responseJson.SetNamedValue(L"name", JsonValue::CreateStringValue(err.Name()));
-  responseJson.SetNamedValue(L"componentStack", JsonValue::CreateStringValue(err.ComponentStack()));
```

Comment:
Did we lose the componentStack?

Reply:
Yeah, but we weren't actually using it apart from previously sending it over the wire. Right now we just recreate and rethrow the error so that Jest will show it, instead of doing anything custom to display it. I did discover that the error message sent to the redbox handler actually gets the component stack embedded into it as well, for some reason, so we still see the component stack as part of the error in the test runner when available.